eng/Packages.Data.props

```diff
@@ -153,6 +153,7 @@
   <ItemGroup Condition="'$(IsExtensionClientLibrary)' == 'true'">
     <PackageReference Update="Microsoft.AspNetCore.DataProtection" Version="2.1.0" />
     <PackageReference Update="Microsoft.Extensions.Azure" Version="1.0.0" />
+    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.1.0-beta.1" Condition="'$(HasReleaseVersion)' == 'false'" />
```

Comment:
@pakrym, Orthogonal but we should consider using the "HasReleaseVersion" property to block folks from depending on the experimental packages in GA packages.

Reply:
The https://github.com/Azure/azure-sdk-for-net/pull/16687/files#diff-42945218ad7165bbb831cc5babc82e241aa0e15004e3699cdf89de8ce2de9c0eR180 should catch it because the Experimental always has a preview version.